backend/src/game/routes.ts

```diff
@@ -70,8 +70,8 @@ router.get(['/og/results/:id', /\/og($|\/.*)/], async (ctx) => {
 
   await ctx.render('results', {
     ...res,
-    dead: formatNumber(data.results.dead),
-    cost: formatNumber(data.results.cost, true, true),
+    dead: formattingService.formatNumber(data.results.dead),
```

Comment:
Have you tested this?

Reply:
For 'cs' (hardcoded now): Moje skóre je 146 193 mrtvých a 30 mld. Kč v nákladech.
For 'en' (only test): Moje skóre je 146,193 mrtvých a Cr 30 mld. v nákladech.